src/main.rs

```diff
@@ -54,6 +54,18 @@ struct Cli {
     /// GGA update period, in seconds. 0 means to never send a GGA
     #[clap(long, default_value = "10")]
     gga_period: u64,
+
+    /// AreaId to be used in generation of CRA message. If this flag is set, ntripping outputs messages of type CRA rather than the default CRA.
```

Comment:
super nit: My OCD is kicking in a bit with `AreaId`, `SolutionID` and 	`Client ID` - could we pick one of these 3 styles?

Reply:
@silverjam the parser does not accept dashes for field names, but I have refactored to <name>_id @joelynch

Reply:
I was joking 😅 -- `/s` is a sarcasm signifier